src/sentry/static/sentry/app/views/settings/components/forms/type.tsx

```diff
@@ -108,10 +108,22 @@ type RangeType = {type: 'range'} & Omit<RangeSlider['props'], 'value'> & {
     value?: Pick<RangeSlider['props'], 'value'>;
   };
 
-type TableType = {
+export type TableType = {
```

Comment:
I couldn't find a way to use these props in the TableField component without exporting like this. Might need to do this when we convert other components as well.